website/content/contributors/_index.md

```diff
@@ -10,51 +10,61 @@ Welcome! Are you interested in contributing to one of CNCF hosted projects? This
+- [Google Season of Docs](https://github.com/cncf/mentoring/tree/master/seasonofdocs) (GSoD)
+- [Outreachy](https://github.com/cncf/mentoring/tree/master/outreachy)
 
-Working groups (WG's) are the community-driven groups with the goal of continuous collaboration in the specific areas. CNCF WG's are created and curated by the CNCF TOC and driven by the community members. CNCF TOC repo provides more [details](https://github.com/cncf/toc/tree/master/workinggroups#cncf-working-groups) on the purpose and goals of WG's, together with the [list of them](https://github.com/cncf/toc/blob/master/README.md#working-groups).
+If you are interested in participating in one of the programs, please check out the [CNCF mentoring repository](https://github.com/cncf/mentoring) for more details.
 
-## Community Engagement
+### Project Opportunities
 
-### Ambassadors
+The CNCF projects are always looking for new contributors. If you're interested in contributing to a CNCF project, check out the following:
 
-[Cloud Native Ambassadors](https://www.cncf.io/people/ambassadors/) (CNAs) are individuals who are passionate about [Cloud Native Computing Foundation](https://www.cncf.io/) technology and projects, recognized for their expertise, and willing to help others learn about the framework and community.
+- [CNCF Landscape](https://landscape.cncf.io/) - a map of all CNCF projects, along with their maturity level and their category
+- [CLOTributor](https://clotributor.dev/) - a tool that helps you find beginner-friendly tasks in CNCF projects
+- [Projects section below](#projects) lists all CNCF projects, with their primary language and brief information on contributing to them
 
-Successful ambassadors are people such as bloggers, influencers, evangelists who are already engaged with a CNCF project in some way including contributing to forums, online groups, community events, etc.
 
-Details on the Ambassadors program, and information on how to join CNCF as an Ambassador is available [here](https://github.com/cncf/ambassadors).
+### TAGs and WGs
 
-### Meetups
+CNCF Technical Advisory Groups (TAGs) oversee and coordinate the interests pertaining to a logical area of needs of end users and/or projects. More details about the CNCF TAGs is available [here](https://github.com/cncf/toc/tree/main/tags).
 
-The Cloud Native Computing Foundation supports the worldwide community of the Cloud Native meetups. They are listed on [meetup.com](https://www.meetup.com/pro/cncf/) website.
+Working groups (WGs) are the community-driven groups with the goal of continuous collaboration in the specific areas. CNCF WGs are created and curated by the CNCF TOC and driven by the community members. CNCF TOC repo provides more [details](https://github.com/cncf/toc/tree/master/workinggroups#cncf-working-groups) on the purpose and goals of WGs, together with the [list of them](https://github.com/cncf/toc/blob/master/README.md#working-groups).
```

Comment:
I think we should remove the bit about WGs. You can see from the [TOC readme](https://github.com/cncf/toc) that there is only 1 standalone working group at the TOC level that is active (serverless). It might be better to say that many of the TAGs have WGs that people can get involved in, which puts the focus on finding a TAG to contribute to and then engaging with WGs within that TAG.

Reply:
Thanks for the feedback. I agree.

I've updated this section now. Can you have a new look?